src/sentry/sentry_metrics/querying/visitors/base.py

```diff
@@ -39,7 +39,7 @@ def _visit_formula(self, formula: Formula) -> TVisited:
         return formula.set_parameters(parameters)
 
     def _visit_timeseries(self, timeseries: Timeseries) -> TVisited:
-        return timeseries
+        raise timeseries
```

Comment:
```suggestion
        return timeseries
```
This was just a conflict since I fixed this in my PR.

Reply:
Changed